templates/clusterclass/nmt-md.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: NutanixMachineTemplate
 metadata:
-  name: "${CLUSTER_CLASS_NAME}-md-nmt"
+  name: "nutanix-quick-start-md-nmt"
 spec:
   template:
     spec:
-      providerID: "nutanix://${CLUSTER_NAME}-m1"
+      providerID: "nutanix://vm-uuid"
```

Comment:
Does this need to be here at all in the CC templates?

Reply:
this is just a place holder and does not require variable. we can explore if we can remove it completely in separate pr. will track it